packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -483,7 +483,7 @@ export class ProfilesUtils {
     public static async promptCredentials(node: IZoweTreeNode): Promise<void> {
         ZoweLogger.trace("ProfilesUtils.promptCredentials called.");
         const mProfileInfo = await globals.PROFILES_CACHE.getProfileInfo();
-        if (!imperative.ProfileInfo.onlyV1ProfilesExist && !mProfileInfo.getTeamConfig().properties.autoStore) {
+        if (!mProfileInfo.getTeamConfig().exists && !mProfileInfo.getTeamConfig().properties.autoStore) {
```

Comment:
We should drop the `!` for the first condition - if the config doesn't exist, we shouldn't check its properties